src/FSharp.Data.GraphQL.Server/Ast.fs

```diff
@@ -1,8 +1,10 @@
-module FSharp.Data.GraphQL.Server.AspNetCore.Ast
+module FSharp.Data.GraphQL.Server.Ast
 
 open System.Collections.Immutable
 open FSharp.Data.GraphQL
 
+/// A list of fields that are reserved for the GraphQL introspection system.
+[<CompiledName "MetaTypeFields">]
```

Comment:
What's the motivation for using a different compiled name?

Reply:
Make it friendly to C# as it is a public API